source/unified-test-format/unified-test-format.rst

```diff
@@ -3,13 +3,13 @@ Unified Test Format
 ===================
 
 :Spec Title: Unified Test Format
-:Spec Version: 1.2.3
+:Spec Version: 1.2.4
```

Comment:
I was on the fence about bumping the minor or patch version, but this would avoid conflicts with the other larger PRs on the spec (both of which are introducing schema changes).